### DISCUSSION TITLE
Failing to install flutter sdk

### DISCUSSION THREAD

**Discussion by Slim-junior** [community/community#88456] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Steps to reproduce
I open visual studio code.
I open the command line and type in flutter and go on flutter new projects then a pop up menu appears after that I go on download sdk and create a folder

Actual results
Failed to clone Flutter: Cloning into 'flutter'...
error: RPC failed; HTTP 400 curl 22 The requested URL returned error: 400
fatal: expected 'packfile'
THIS IS THE RESULT

**Comment by mishmanners** [comment]
I've retagged this as bug so the appropriate team can find it. Thanks for reaching out to us 😄

**Reply by Slim-junior** [reply]
thank you

**Comment by BrillianceOsprey** [comment]
I think you need to install Git first.
Here is link 
https://git-scm.com/

**Reply by DEVigodE** [reply]
yes

**Reply by Slim-junior** [reply]
already did now started working...only problem is launching the app using the emulator which is now not working

**Comment by DEVigodE** [comment]
https://www.youtube.com/watch?v=VFDbZk2xhO4

**Reply by Slim-junior** [reply]
Thank you very much this video helped i managed to install flutter however the other problem im having is launching an app on the virtual machine/emulator if i try to launch it on the chrome website its going through but on the virtual device its not working
